manifest.json

```diff
@@ -88,25 +96,25 @@
         "targetDirPrefix": "callbacks-"
       },
       "beautifulSoup": {
-        "version": "4.7.1",
+        "version": "4.9.0",
         "downloadFormat": "tar",
-        "url": "https://pypi.python.org/packages/80/f2/f6aca7f1b209bb9a7ef069d68813b091c8c3620642b568dac4eb0e507748/beautifulsoup4-4.7.1.tar.gz#md5=321d73746d06bd9a8fcc3787b3fb7598",
+        "url": "https://files.pythonhosted.org/packages/3b/e4/7cfc641f11e0eef60123912611a5c9ee7d4638da7325878b695b9ae4bb6f/beautifulsoup4-4.9.0.tar.gz#md5=54e86045e37b578036f34b0e4927c85f",
         "tarRootDirPrefix": "beautifulsoup4-",
         "rootDirPrefix": "beautifulsoup4-",
         "targetDirPrefix": "beautifulsoup4-"
       },
       "mutagen": {
-        "version": "1.42.0",
+        "version": "1.43.0",
         "downloadFormat": "tar",
-        "url": "https://files.pythonhosted.org/packages/30/4c/5ad1a6e1ccbcfaf6462db727989c302d9d721beedd9b09c11e6f0c7065b0/mutagen-1.42.0.tar.gz",
+        "url": "https://files.pythonhosted.org/packages/aa/fd/ed738775442e3614849fefdf41417d7bff3ccb010d49c8f729432cc3c1e5/mutagen-1.43.0.tar.gz#md5=3154511015860685b3882c068258ae65",
```

Comment:
Any specific reason for adding #md5... at the end (diito above)

Reply:
It verifies that the archive is correct, it was already used in the other libs that we use.